addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -6420,6 +6556,33 @@ registry.ImageTools = ImageHandlerOption.extend({
             "img_shape_transform_rotate_x_opt", "img_shape_transform_rotate_y_opt"].includes(params.name)) {
             return this._isTransformableShape();
         }
+        if (widgetName === "hover_effect_none_opt") {
+            // The hover effects are removed with the "WebsiteAnimate" animation
+            // selector so this option should not be visible.
+            return false;
+        }
+        if (params.optionsPossibleValues.setImgShapeHoverEffect) {
+            const imgEl = this._getImg();
+            return imgEl.classList.contains("o_animate_on_hover") && !this._isDeviceShape() && !this._isAnimatedShape();
+        }
+        // If "Description" or "Tooltip" options.
+        if (["alt", "title"].includes(params.attributeName)) {
+            const imgEl = this._getImg();
+            return !imgEl.matches("[data-oe-type='image'] > img, [data-oe-xpath]");
```

Comment:
This is equivalent to calling `isImageSupportedForStyle` or something like that, should be changed.